modules/render.js

```diff
@@ -21,7 +22,8 @@ class Render {
   }
 
   static clearField() {
```

Comment:
- [ ] it would be better if you use the es6 `arrow function` instead of the `normal function` in order to practice to work with both